src/main/webapp/js/userMapData.json

```diff
@@ -132,16 +142,20 @@
 ["City College","Greece"],
 ["MIO-ECSDE","Greece"],
 ["Colegio Americano del Sur","Guatemala"],
+["CCC Rotary Secondary School","Hong Kong"],
 ["HKU SPACE Community College","Hong Kong"],
 ["Queen Elizabeth School","Hong Kong"],
 ["The Hong Kong Polytechnic University","Hong Kong"],
 ["University of Hong Kong","Hong Kong"],
+["Codecool","Hungary"],
 ["All India Institute of Medical Sciences","India"],
+["Bharati Vidypeeth College of Engineering, New Delhi","India"],
```

Comment:
I think this one should be Vidy**a**peeth and not Vidypeeth. Please re-check all the spellings once